CGAL_ImageIO/src/CGAL_ImageIO/gis_impl.h

```diff
@@ -143,7 +143,7 @@ int writeGis( char *name, _image* im) {
 		if ( j<n && i<size ) sprintf( str+strlen(str), " " );
 	      }
 	      sprintf( str+strlen(str), "\n" );
-	      res = ImageIO_write( im, str, strlen( str )  );
+	      res = static_cast<int>(ImageIO_write( im, str, strlen( str )));
 	      if ( res  <= 0 ) {
```

Comment:
Fixed in commit 3b61a9d611c26b25c70de8fa0b8ddd1f9c94e630.
It's not perfect, but the way errors are handled in the code is a bit hard to deal with...